lib/json_api_assert.ex

```diff
@@ -55,6 +55,35 @@ defmodule JsonApiAssert do
     do: raise ExUnit.AssertionError, "jsonapi object not found"
 
   @doc """
+  Asserts that a valid meta object exists in the payload
+
+    payload
+    |> assert_meta([:copywrite, :authors])
```

Comment:
Hey Chris.  Are you suggesting I add the examples header? I was following the style of the existing docs.